test/index.js

```diff
@@ -1950,4 +1950,88 @@ describe('normalizr', function () {
     }).should.throw();
   });
 
+  it('can assign parent id on child entity', function () {
+    var article = new Schema('articles'),
+      comment = new Schema('comments', { assignParentId: true }),
```

Comment:
Didn't cover this case, but a suggestion is that when there are multiple parents the child is assigned an array of parent ids.